### PR TITLE
Fix versions for PrivateNetworkAccessKillswitch [production]

### PR DESCRIPTION
**Pull request by atuchin-m**
Resolves https://github.com/brave/brave-variations/issues/1027
PR in main: https://github.com/brave/brave-variations/pull/1028

### CODE REVIEW

**Review by goodov — APPROVED** [review]

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers.

**Pull request merged**
